airflow/providers/apache/sqoop/hooks/sqoop.py

```diff
@@ -113,6 +113,9 @@ def popen(self, cmd: list[str], **kwargs: Any) -> None:
                 raise AirflowException(f"Sqoop command failed: {masked_cmd}")
 
     def _prepare_command(self, export: bool = False) -> list[str]:
+        if "?" in self.conn.host:
```

Comment:
is this condition applicable ONLY for sqoop connection ?

Reply:
Yep. This is specificlaly for sqoop.